UI/Web/src/app/on-deck/on-deck.component.ts

```diff
@@ -63,10 +63,10 @@ export class OnDeckComponent implements OnInit {
     this.loadPage();
   }
 
-  updateFilter(data: SeriesFilter, isFirst) {
-    this.filter = data;
+  updateFilter(event: FilterEvent) {
+    this.filter = event.filter;
     const page = this.getPage();
-    if (page === undefined || page === null || isFirst !== true) {
+    if (page === undefined || page === null || event.isFirst !== true) {
```

Comment:
We can just change this to !event.isFirst as it's a boolean.

Reply:
resolved with 7ab4709